java/rocksjni/sst_file_writerjni.cc

```diff
@@ -74,16 +74,16 @@ void Java_org_rocksdb_SstFileWriter_open(JNIEnv *env, jobject jobj,
 
 /*
  * Class:     org_rocksdb_SstFileWriter
- * Method:    add
+ * Method:    put
  * Signature: (JJJ)V
  */
-void Java_org_rocksdb_SstFileWriter_add(JNIEnv *env, jobject jobj,
```

Comment:
I think this will break the Java build, as the code in SstFileWriter.java uses this.